### PR TITLE
Add more mistral examples to integration test

### PR DESCRIPTION
**Pull request by m4dcoder**

### CODE REVIEW

File: contrib/examples/actions/workflows/mistral-env-var.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+examples.mistral-env-var:
+    description: A basic workflow that illustrates how to get the workflow's env vars.
+    type: direct
+    output:
+        stdout: <% $.stdout %>
+    tasks:
+        task1:
+            action: core.local
+            input:
+                cmd: echo <% env().st2_execution_id %>
```
Comment (dzimine) [inline contrib/examples/actions/workflows/mistral-env-var.yaml]: I suggest we show the example with the URL. It's a real use case people are asking and we use broadly. Hardcoding IP is fine, they'll get it. Like `https://172.168.50.11/#/history/<% env().st2_execution_id %>/general`

This gives an idea, and answers a bunch of know-how.

File: contrib/examples/actions/workflows/mistral-env-var.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+            publish:
+                stdout: <% $.task1.stdout %>
```
Comment (dzimine) [inline contrib/examples/actions/workflows/mistral-env-var.yaml]: I would also publish env to make it ready for an exploring user to see what else good inside env. 
`env: <% env() %> `

**Comment by dzimine** [comment]
not directly relevant, but is there a reason to turn "run all mistral examples" action chain to "run some mistral examples" ?

File: contrib/examples/actions/workflows/mistral-env-var.yaml (hunk 1 above)
Comment (m4dcoder) [inline contrib/examples/actions/workflows/mistral-env-var.yaml]: Sure.

File: contrib/examples/actions/workflows/mistral-env-var.yaml (hunk 2 above)
Comment (m4dcoder) [inline contrib/examples/actions/workflows/mistral-env-var.yaml]: Ok.

**Comment by m4dcoder** [comment]
Yes, the AC you are referring to is there to test basic mistral functionality.  All the examples are tested in itests for CICD purposes.

**Pull request merged**